buildbot-host/buildmaster/launch.sh

```diff
@@ -15,4 +15,9 @@ fi
 docker container stop buildmaster
 docker container rm buildmaster
 
-docker container run -it --name buildmaster --restart no --volume /var/lib/repos:/var/lib/repos --mount source=buildmaster,target=/var/lib/buildbot/masters/default/persistent --network buildbot-net --publish 8010:8010 --publish 9989:9989 openvpn_community/buildmaster:$TAG
+# interactive
+DOCKER_OPTS=-it
+# detached
+DOCKER_OPTS=-d
```

Comment:
How about having a sane default and allowing override from command-line, e.g. like this:

    DOCKER_OPTS="${DOCKER_OPTS:--d}"

So you could do

    DOCKER_OPTS=-it launch.sh